src/components/Garden/Stake/StakingMovements.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-redeclare */
+/* eslint-disable react/jsx-key */
```

Comment:
Why we need disable jsx-key, its kind unsafe do not have key property to render list?

Reply:
Its a stupid error actually, but its regarding this.

<img width="882" alt="Captura de ecrã 2022-04-05, às 18 58 00" src="https://user-images.githubusercontent.com/13184907/161819957-0dedf9b1-ce23-46e0-ae9f-fd52cd5b2302.png">

Reply:
Okay, i think we need add key in <time and the others <divs or disable to that specific part the code instead the whole file, WDYT?

Reply:
and if I take that it shows this 
<img width="1023" alt="Captura de ecrã 2022-04-05, às 19 01 14" src="https://user-images.githubusercontent.com/13184907/161820450-401a83e2-6036-407c-8c5a-8165b589e695.png">

Reply:
yep but its super strange that don't you think?

Reply:
Its not a rendered list per se, is just a return array of elements.

Reply:
What is strange? The error? Its because that elements are inside the array, so react need key to in case some object its removed dynamic that list it re-renders correctly.

Because that the recommended its put some unique key in that.

Reply:
I will change to add the keys :)